assets/src/stories-editor/helpers/index.js

```diff
@@ -1,3 +1,5 @@
+/* global _ */
```

Comment:
Better to use `import { each } from 'lodash';` for that. Webpack maps that to window.lodash automatically. This way you also get autocompletion in the IDE.

Although for this use case just using `settings.transforms.from.forEach()` should be fine too.